### ISSUE TITLE
Overwrite edited images option

### ISSUE THREAD

**Issue by Yanpas** [SimpleMobileTools/Simple-Gallery#1054]
Most often i need to crop some screenshot

**Comment by Yanpas** [comment]
Yes, it suggests a name PHOTO_1. And I need to remove two symbols to overwrite.

**Comment by 07416** [comment]
> Yes, it suggests a name PHOTO_1. And I need to remove two symbols to overwrite.

You have to remove two characters with each image? Is there an automated option?

**Comment by Yanpas** [comment]
Yes, redownloaded app and still can't get why the issue was closed
